_data-prepper/trace-tuning.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Trace tuning
+nav_order: 12
+---
+
+# Trace tuning
+
+Starting in version 0.8.x, Data Prepper for trace analytics supports both vertical and horizontal scaling. You can adjust the size of your single Data Prepper instance to meet your workload's demands and scale vertically. 
+
+You can scale horizontally by deploying multiple Data Prepper instances to form a cluster by using the [Core Peer Forwarder](https://github.com/opensearch-project/data-prepper/blob/main/docs/peer_forwarder.md). This enables Data Prepper instances to communicate with instances in the cluster, and is required for horizontally-scaling deployments.
+
+## Scaling tips
+
+The following sections describe useful tips for scaling. We recommend that you modify parameters based on the requirements. We also recommend that you monitor the Data Prepper host metrics and OpenSearch metrics to ensure the configuration is working as expected.
+
+### Buffer
+
+The total number of trace requests that Data Prepper is processing is equal to sum of `buffer_size` in `otel-trace-pipeline` and `raw-trace-pipeline`. The total number of trace requests to OpenSearch is equal to the product of `batch_size` and `workers` in `raw-trace-pipeline`.
+
+We recommend that you use the following parameters for buffer settings:
+ * Have the same `buffer_size` value in `otel-trace-pipeline` and `raw-trace-pipeline`
+ * That the values `buffer_size` >= `workers` * `batch_size` in the `raw-trace-pipeline`
+ 
+
+### Workers 
+
+The `workers` setting determines the number of threads that are used by Data Prepper to process requests from the buffer. We recommend that you set `workers` based on the CPU utilization. This value can be higher than available processors, as Data Prepper spends significant input/output time when sending data to OpenSearch.
+
+### Heap
+
+Configure the heap of Data Prepper by setting the `JVM_OPTS` environmental variable. We recommend that you set the heap value to a minimum of `4` * `batch_size` * `otel_send_batch_size` * `maximum size of indvidual span`.
+
+As mentioned in the [setup setpes](trace_setup.md#opentelemetry-collector), set `otel_send_batch_size` to a value of `50` in your opentelemetry collector configuration.
```

Comment:
Is this the right spelling?

"setup setpes"

Reply:
Both of these points are correct.